### DISCUSSION TITLE
DAILY 14/05

### DISCUSSION THREAD

**Discussion by AhumadaCeleste** [AhumadaCeleste/DATA-WAVE#76] [General]
HOY 13/05: COMPLETE EL CRUD DE INSTITUTO.

HOY 14/05:TERMINADO EL LOGUIN CON PASWORD EN  BACKEND, IMPLEMENTANDO EL LOGUIN EN FRONT.

MAÑANA: 15/05: TERMINAR DE DAR LOS ULTIMOS AJUSTES A LA PAGINA DE INSPECTOR. CORREGIR LA GRÁFICA DE LOS BOTONES, EMPAREJAR EL COLOR DE BG. HOVER, COLOR DE BOTONES MODIFICAR Y ELIMINAR PARA YA  DEJARLOS LISTOS.

**Comment by cecipiombo** [comment]
Creo que logré entrar y encontré las daily.
me gustaría poder ver tu proyecto, asique si querés cargar en esta tarea el video para mostrarme los avances, sería bueno. Y de paso, vamos utilizando la herramienta como seguimiento.

**Comment by AhumadaCeleste** [comment]
Profe: adjunto vinculo con video del seguimiento. https://drive.google.com/file/d/1oWm1jN36WCHoUj3K3tiWDlLgHIYnA80K/view?usp=sharing

**Reply by cecipiombo** [reply]
perfecto el video Celeste.
entendí todo
